scripts/src/java/org/oppia/android/scripts/license/MavenDependenciesListCheck.kt

```diff
@@ -179,44 +178,7 @@ class MavenDependenciesListCheck(
 
   private fun printDependenciesList(dependencyList: List<MavenDependency>) {
     dependencyList.forEach { dep ->
-      println(
-        """
-        artifact_name: "${dep.artifactName}"
-        artifact_version: "${dep.artifactVersion}"
-        """.trimIndent()
-      )
-      dep.licenseList.forEach { license ->
-        printLicense(license)
-      }
-      println()
-    }
-  }
-
-  private fun printLicense(license: License) {
-    println(
-      """
-      license {
-        license_name: "${license.licenseName}"
-        original_link: "${license.originalLink}"
-      """.trimIndent()
-    )
-    when (license.verifiedLinkCase) {
-      License.VerifiedLinkCase.SCRAPABLE_LINK -> println(
-        """
-          scrapbale_link: "${license.scrapableLink.url}"
-        """.trimIndent()
-      )
-      License.VerifiedLinkCase.EXTRACTED_COPY_LINK -> println(
-        """
-          extracted_copy_link: "${license.extractedCopyLink.url}"
-        """.trimIndent()
-      )
-      License.VerifiedLinkCase.DIRECT_LINK_ONLY -> println(
-        """
-          direct_link_only: "${license.directLinkOnly.url}"
-        """.trimIndent()
-      )
+      println(dep)
```

Comment:
Can you print the TextFormat version of the dep, instead? I don't think there's any guarantee that the Message.toString() method prints a parsable text proto (though it seems to be close).